assets/sass/cds/_mixins.scss

```diff
@@ -2,8 +2,8 @@
 
 @mixin mobile_only  { @media ( max-width: 767px )  { @content; } }
 @mixin tablet       { @media ( min-width: 768px )  { @content; } }
+// @mixin tv           { @media ( min-width: 900px ) { @content; } }
```

Comment:
Can we remove this one as well instead of commenting out?

Reply:
oh oops I meant to. Thank you for catching that!!